Makefile

```diff
@@ -21,15 +21,18 @@ terraform-staging-apply: ## Apply infrastructure changes for staging
 clean: ## Remove compiled files
 	rm -rf dist/*
 
-build: clean dist/index.html dist/bundle.js ## Compile the app
+build: clean dist/index.html dist/assets dist/webpack ## Compile the app
 
-dist/bundle.js:
+dist/webpack:
```

Comment:
Can you change this to be like master?

Also there's no file at `dist/webpack` so this would need an entry in `.PHONY`.